src/OSPSuite.Presentation/Binders/ObservedDataDragDropBinder.cs

```diff
@@ -24,6 +25,25 @@ public IReadOnlyList<DataRepository> DroppedObservedDataFrom(IDragEvent e)
          return getObservedDataNodesFrom(e).Select(observedDataNode => observedDataNode.Tag.Subject).ToList();
       }
 
+      public Cache<string, List<DataRepository>> DroppedObservedDataWithFolderPathFrom(IDragEvent e)
```

Comment:
What's with the signature? Why do you need a cache with an Id? What does it represent? How is this being used?
Isn't it really a list of list?

Reply:
It is the ID of the parent node for internal use, but yes you are right, we do not need to give it as a Cache from the public function. I corrected this